constrained-voucher.mkd

```diff
@@ -410,6 +411,14 @@ If the Pledge included in its request an Accept Option for only the TBD287 ("app
 
 If the current authenticated client is an EST-coaps client that was already enrolled in the domain, and the Registrar is configured to assign this client to a new domain CA trust anchor during the next EST re-enrollment procedure, then the Registrar MUST respond with the new domain CA certificate in case the client performs the "CA Certificates request" (/crts) with an Accept Option for TBD287 only. This signals the client that a new domain is assigned to it. The client follows the procedure as defined in {{brski-est-extensions-estclient}}.
 
+### CoAP responses {#brski-est-responses}
+
+{{RFC8995, Section 5.6}} defines a number of HTTP response codes that the Registrar is to return when certain conditions occur.
+
+The 403,404,406 and 415 response codes map directly to CoAP codes 4.03, 4.04, 4.06 and 4.15.
+
+The 202 Retry process needs to map to a CoAP Separate Response as detailed in {{RFC7252, Section 5.2.2}}.
```

Comment:
Here we should align with ace-coap-est https://datatracker.ietf.org/doc/html/draft-ietf-ace-coap-est#section-5.7 Delayed Responses mechanism!  Although this uses the 5.03 server-error code instead of a success code, in CoAP the meaning of 5.03 is most similar to the HTTP 202 mechanism. And we keep the mechanisms between constrained BRSKI/EST aligned as well!